src/ansible_navigator/runner/command_async.py

```diff
@@ -0,0 +1,40 @@
+""" Herewith lies the ability to invoke ansible-runner
+in an async fashion. A queue is provided and ansible-runner
+uses pexpect to parse standout and error from the command run
+and populates the queue with messages
+"""
+
+from queue import Queue
+from ansible_runner import run_command_async
+
+from .command_base import CommandBase
+
+
+class CommandAsync(CommandBase):
+    # pylint: disable=too-few-public-methods
+    # pylint: disable=too-many-instance-attributes
+    """a runner async wrapper"""
+
+    def __init__(self, executable_cmd: str, queue: Queue, **kwargs):
+        """class to handle arguments of ``run_command_async`` interface for ``ansible-runner``.
+           For common arguments refer documentation of ``CommandBaseRunner`` class
```

Comment:
This docstring is in violation of PEP 257 and it also provides invalid information. First of all, it's not a class but an initializer. Then, a docstring title must be a full sentence (with a capital letter at the beginning and a trailing period at the end). The first line should fit in 72 chars and be separated from prose with an empty line.
```suggestion
        """Initialize the arguments for the ``run_command_async`` interface of ``ansible-runner``.

        For common arguments refer to the documentation of the ``CommandBaseRunner`` class.
```

Reply:
Fixed in #630